Examples/Python/ellipsoid.py

```diff
@@ -283,9 +297,17 @@ def Run_Pipeline(args):
     # Run multiscale optimization unless single scale is specified
     if not args.use_single_scale:
         parameter_dictionary["use_shape_statistics_after"] = 32
-    # Execute the optimization function
+
+     # If in mesh mode optimize on meshes
```

Comment:
The `if args.mesh_mode` will be completely abstracted when the consolidation-into-a-single-function request is complete. There are six of these copies, each 15 lines long, split over two disparate sections in each test in which they're used, all reducible to a single function.